src/index.html

```diff
@@ -6,14 +6,463 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Title</title>
+    <title>WITCH OF MOUNTAIN WINDS</title>
+    <link
+      rel="shortcut icon"
+      href="favicon.svg"
+      type="image/x-icon"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page-content">
+    <header
+      class="header"
+      id="home"
+    >
+      <nav class="header__navigation navigation">
+        <a
+          href="#menu"
+          class="navigation__item"
+        >
+          <img
+            src="./images/icons/menu-icon.svg"
+            alt="menu-icon"
+            class="navigation__icon navigation__icon--menu"
+          />
+        </a>
+
+        <a
+          href="#home"
+          class="navigation__item"
+        >
+          <img
+            src="./images/icons/logo-icon.svg"
+            alt="logo-icon"
+            class="navigation__icon"
+          />
+        </a>
+
+        <a
+          href="#shop"
+          class="navigation__item"
+        >
+          <img
+            src="./images/icons/shop-icon.svg"
+            alt="shop-icon"
+            class="navigation__icon"
+          />
+        </a>
+      </nav>
+
+      <img
+        src="./images/header-picture.jpg"
+        alt="header-picture"
+        class="header__picture"
+      />
+
+      <h1 class="header__title">Brand of eco-cosmetics and magic candles</h1>
+
+      <p class="header__text">20% of sales at a wild bird shelter</p>
+
+      <a
+        href="#shop"
+        class="button--type1 header__button--type1"
```

Comment:
`button--type1` it's modifier but it's used like block 
(https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-a-modifier-without-the-belonging-class) 

```suggestion
        class="button button-type1 header__button-type1"
```
or
```suggestion
        class="button button--type1 header__button"
```
or
```suggestion
        class=""header__button header__button--type1"
```